skcosmo/utils/_pcovr_utils.py

```diff
@@ -72,8 +132,8 @@ def pcovr_covariance(
                 random_state=random_state,
             )
 
-        UC = UC.T[:, vC > rcond]
-        vC = vC[vC > rcond]
+            UC = UC.T[:, (vC ** 2) > rcond]
+            vC = vC[(vC ** 2) > rcond]
```

Comment:
Could you add a test hitting this code path? Even something simple with toy X/Y matrices.

Reply:
@bhelfrecht, codecov is still not seeing these lines in the coverage report, which is really weird since you explicitly seeded the rng ... Maybe different version of numpy have different RNG? CI is using numpy == 1.20.2.

Reply:
Just fixed it -- there was a typo in the test that made it bypass the targeted code